test/parallel_api/dynamic_selection/test_dynamic_load_policy_inline.pass.cpp

```diff
@@ -0,0 +1,59 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <iostream>
+#include "oneapi/dpl/dynamic_selection"
+#include "support/test_dynamic_selection_utils.h"
```

Comment:
This includes a different file than the load_policy_sycl test. And this file seems to be the only includer of the file. So why put that header in the support directory?